src/tree/serviceExplorer.ts

```diff
@@ -11,6 +11,12 @@ import { ServiceDataProvider } from './serviceDataProvider';
 import { WatchUtil, FileContentChangeNotifier } from '../util/watch';
 
 const kubeConfigFolder: string = path.join(Platform.getUserHomePath(), '.kube');
+const kubeconfigSplit: string[] = process.env.KUBECONFIG.split('/');
```

Comment:
I think this needs to use `path.delimiter` as `process.env.KUBECONFIG.split(path.delimiter);`

Reply:
good catch